R/local.R

```diff
@@ -65,7 +65,7 @@
 #'   cat("\n")
 #' })
 local_test_context <- function(.env = parent.frame()) {
-  withr::local_envvar(TESTTHAT = "true", .local_envir = .env)
+  withr::local_envvar("_R_CHECK_BROWSER_NONINTERACTIVE_" = "true", TESTTHAT = "true", .local_envir = .env)
```

Comment:
I think we still want to gate this behind `!interactive()` because it useful to use `browser()` when interactively running tests.

Reply:
The environment variable only has an effect in non-interactive sessions so "technically" it's still okay to set here. (It also makes sure it's set if you run R as a sub-process from an interactive parent process, which could potentially pop up in some tests)

Reply:
Oh duh.